routes/index.go

```diff
@@ -14,90 +14,38 @@ import (
 
 	"github.com/stakwork/sphinx-tribes/auth"
 	"github.com/stakwork/sphinx-tribes/db"
-	"github.com/stakwork/sphinx-tribes/frontend"
 	"github.com/stakwork/sphinx-tribes/handlers"
 )
 
 // NewRouter creates a chi router
 func NewRouter() *http.Server {
 	r := initChi()
 
-	r.Get("/ping", func(w http.ResponseWriter, r *http.Request) {
-		w.WriteHeader(http.StatusOK)
-		json.NewEncoder(w).Encode("pong")
-	})
-
-	r.Group(func(r chi.Router) {
-		r.Get("/", frontend.IndexRoute)
-		r.Get("/static/*", frontend.StaticRoute)
-		r.Get("/manifest.json", frontend.ManifestRoute)
-		r.Get("/favicon.ico", frontend.FaviconRoute)
-	})
-	r.Group(func(r chi.Router) {
-		r.Get("/t/static/*", frontend.StaticRoute)
-		r.Get("/t/manifest.json", frontend.ManifestRoute)
-		r.Get("/t/favicon.ico", frontend.FaviconRoute)
-		r.Get("/t/{unique_name}", frontend.IndexRoute)
-		r.Get("/t", frontend.IndexRoute)
-	})
-	r.Group(func(r chi.Router) {
-		r.Get("/p/static/*", frontend.StaticRoute)
-		r.Get("/p/manifest.json", frontend.ManifestRoute)
-		r.Get("/p/favicon.ico", frontend.FaviconRoute)
-		r.Get("/p/{pubkey}", frontend.IndexRoute)
-		r.Get("/p/{pubkey}/offer", frontend.IndexRoute)
-		r.Get("/p/{pubkey}/badges", frontend.IndexRoute)
-		r.Get("/p/{pubkey}/wanted", frontend.IndexRoute)
-		r.Get("/p/{pubkey}/usertickets", frontend.IndexRoute)
-		r.Get("/p", frontend.IndexRoute)
-		r.Get("/b", frontend.IndexRoute)
-		r.Get("/tickets", frontend.IndexRoute)
-		r.Get("/leaderboard", frontend.IndexRoute)
```

Comment:
missing this route

Reply:
ok nice

fixed in [02e321c](https://github.com/stakwork/sphinx-tribes/pull/527/commits/02e321c22589ecf4988252ba96f9d67b46d497e2)